ui/widgets/selectmenu.js

```diff
@@ -119,15 +119,12 @@ return $.widget( "ui.selectmenu", [ $.ui.formResetMixin, {
 		this._addClass( this.button, "ui-selectmenu-button ui-selectmenu-button-closed",
 			"ui-button ui-widget" );
 
-		icon = $( "<span>" ).prependTo( this.button );
-		space = $( "<span> </span>" );
-		this._addClass( space, "ui-selectmenu-icon-space" );
```

Comment:
Looks like this class isn't used anymore. Can you also do a PR for https://github.com/jquery/api.jqueryui.com/blob/1-12/entries/selectmenu.xml#L44 ?